test/Atmos/EDMF/bomex_edmf.jl

```diff
@@ -267,13 +267,19 @@ function config_diagnostics(driver_config, timeend)
         boundaries;
         axes = axes,
     )
-    dgngrp = setup_dump_state_diagnostics(
+    ds_dgngrp = setup_dump_state_diagnostics(
         SingleStackConfigType(),
         interval,
         driver_config.name,
         interpol = interpol,
     )
-    return ClimateMachine.DiagnosticsConfiguration([dgngrp])
+    dt_dgngrp = setup_dump_tendencies_diagnostics(
```

Comment:
How is this getting exercised?

Reply:
It's only added here for evaluation (could put it somewhere else) and I wasn't intending to put it somewhere it would be exercised on CI -- we don't exercise every diagnostic group (e.g. `Dump*`) because they dump quite a bit of output and there's not a lot of logic to test. But I'm happy to do so if you feel it's necessary.